README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+
+# hw01
+
+My name is **Bingxin** (Janssie) Zhu. In this course, I  want to 
+
+* Learn how to code using R
+* Conduct data anaglsis using R 
+
+#### My research interests 
```

Comment:
Completely a stylistic comment, but it might be useful to use a slightly bigger header. Though I appreciate the aesthetic elements of smaller font sizes, it seems to be roughly the same size as the bolded body text

Reply:
I agree with the above comment. Using fewer `#` will make your header look better